projects/v2/src/app/static/config.ts

```diff
@@ -36,6 +53,17 @@ export const SHEET_CONFIG: SheetDetails[] = [
       width: 1000,
       height: 1400,
     },
+    version: [
+      {
+        value: 'spleen-v1.0',
+        viewValue: 'v1.0',
+        csvUrl: `${version1Url}ASCT-B_VH_Spleen.csv`
+      },
+      {
+        value: 'spleen-v1.5',
+        viewValue: 'v1.5',
```

Comment:
Unofficial/draft tables should be called latest or draft (need to check with @katyb her preferences). They will only get a version number once they are finalized, given a DOI, and saved as CSV like v1.0.